docs/testing/testing-with-puppeteer.md

```diff
@@ -82,6 +82,16 @@ integration](../testing/continuous-integration.md):
   Integration (CI) services because small races are amplified in those
   environments; this often explains failures in CI that cannot be
   easily reproduced locally.
+* Does the test fail when you are typing (filling the form) on modal? If so,
+  the problem is likely that the modal was focused after it starts typing.
+  Puppeteer starts typing after focusing on the text field, sending keystrokes
+  one after another. So, calling `.focus` on modal after it starts
```

Comment:
Specifically, we mean "So, if application code calls focus on a modal after the Puppeteer test has started typing ...", right?